sycl/test/sub_group/load_store.cpp

```diff
@@ -97,10 +105,12 @@ template <typename T> void check(queue &Queue) {
       }
     }
 
+    using TT = typename std::conditional<std::is_same<T, aligned_half>::value,
+                                        wa_half, T>::type;
```

Comment:
Strange indent.
Actually all the `using TT` around...

Reply:
Look like I forgot to apply clang-format for this change. Will do.